django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -52,6 +62,58 @@ def get_queryset(self) -> QuerySet[TeamMember]:
         )
 
 
+class CyberstormTeamAddMemberRequestSerialiazer(serializers.Serializer):
+    user = serializers.CharField(max_length=User._meta.get_field("username").max_length)
```

Comment:
I would recommend using `username` as the name for fields that explicitly want to handle the `username` property of the user objects, and saving `user` for scenarios where we're passing the actual user object around. Otherwise we'll end up with ambiguity between different API endpoints where the same `user` field name is used to represent all kinds of different data.

Also applies to the response serializer